microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,114 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+When you select the Address bar, a signal is sent to your default search provider.  The signal tells the provider to prepare suggestions.  The typed characters and search queries aren't sent to Microsoft unless your search provider is Bing.
 
-<!-- The Privacy settings have been updated from what is described here -->
 
-If Microsoft Edge detects typing in the address bar that might contain sensitive information, it does not send the typed text. Sensitive information includes authentication credentials, local file names, or URL data that is normally encrypted.
+<!-- ------------------------------ -->
+#### Controlling whether typed characters are sent
+
+You can control whether typed characters are sent to your default search provider.  To enable sending search and site suggestions by using your typed characters in the Address bar to your default search provider:
+
+1. Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**, and then scroll down to the **Services** section.
+
+1. Click the **Address bar and search** section.  The **Address bar and search** settings page<!-- edge://settings/search --> opens.
+
+1. Click the **Search suggestions and filters** section.  The **Search suggestions and filters** settings page<!-- edge://settings/searchFilters --> opens.
+
+1. Use the toggle **Show me search and site suggestions using my typed characters**:
+
+   ![The toggle in Edge Settings for "Show me search and site suggestions using my typed characters"](./index-images/search-suggestions-typed-chars.png)
+
+   If you turn off that toggle, your typed characters are no longer sent to your default search provider, but your search queries are still sent to your default search provider, to provide search results.<!--per ui: "If you turn this off, you'll only see suggestions from your favorites and history. Typed characters won't be sent to the search engine used in the address bar." -->
+
+
+<!-- ------------------------------ -->
+#### Microsoft Search in Bing
+
+If you're signed into Microsoft Edge using your work or school account (Microsoft Entra ID account), your typed-in characters are also sent to Microsoft Search in Bing, which returns work- or school-related results and cross-device suggested queries to you.
```

Comment:
Are they sent to bing in addition to your default search provider?

Reply:
added expanded hidden todo comment:
```
If you're signed into Microsoft Edge using your work or 
school account (Microsoft Entra ID account), your typed-in 
characters are also<!--todo: also, in addition to what?  in 
addition to your default search provider?  state what's being 
added to, or delete "also"--> sent to Microsoft Search in Bing, 
which returns work- or school-related results and cross-device 
suggested queries to you.
```

Reply:
handling via other [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3224/files/e51a9da7823f9c97b0cb268d16324930b4fcd534#r1678097658)